src/components/button/types.ts

```diff
@@ -24,6 +24,7 @@ export type ButtonCommonProps = Omit<BaseButtonProps, "kind" | "shape" | "size">
   isLoading?: boolean;
   className?: string;
   role?: AriaRole;
+  tabIndex?: number;
```

Comment:
I think this is not the best way to add props one by one here. Better use `Component props<"button">`

Reply:
updated it